cfme/tests/ssui/test_ssui_myservice.py

```diff
@@ -68,7 +68,10 @@ def test_retire_service(appliance, setup_provider, context, order_catalog_item_i
 def configure_vmware_console_for_test(appliance, provider):
```

Comment:
**REQUIRED:** Let's do this as an uncollect please, skip implies the test should run and is valid for the configuration, it's not.

Reply:
Sure, thank you for the hint.